docs/docs/custom-actions.mdx

```diff
@@ -2,85 +2,51 @@
 id: custom-actions
 sidebar_label: Custom Actions
 title: Custom Actions
+abstract: A custom action can run any code you want, including API calls, database queries etc. They can turn on the lights, add an event to a calendar, check a user's bank balance, or anything else you can imagine.
 ---
 
-An action can run any code you want. Custom actions can turn on the lights,
-add an event to a calendar, check a user's bank balance, or anything
-else you can imagine.
-
-The `Action` class is the base class for any custom action. It has two methods
-that both need to be overwritten, `name()` and `run()`.
-
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="custom-action-example"></a>
-
-In a restaurant bot, if the user says “show me a Mexican restaurant”,
-your bot could execute the action `ActionCheckRestaurants`,
-which might look like this:
-
-```python
-from rasa_sdk import Action
-from rasa_sdk.events import SlotSet
-
-class ActionCheckRestaurants(Action):
-   def name(self) -> Text:
-      return "action_check_restaurants"
-
-   def run(self,
-           dispatcher: CollectingDispatcher,
-           tracker: Tracker,
-           domain: Dict[Text, Any]) -> List[Dict[Text, Any]]:
-
-      cuisine = tracker.get_slot('cuisine')
-      q = "select * from restaurants where cuisine='{0}' limit 1".format(cuisine)
-      result = db.query(q)
-
-      return [SlotSet("matches", result if result is not None else [])]
+For details on how to implement a custom action, see the [SDK documentation](./running-action-server.mdx).
+Any custom action that you want to use in your stories should be added into the
+actions section of your [domain file](./domain.mdx).
```

Comment:
Somewhat random -- now that you can split the domain, we should probably stop referring to it as the domain file, and rather just the domain 🤔